apicapi/apic_manager.py

```diff
@@ -575,7 +576,7 @@ def ensure_bd_created_on_apic(self, tenant_id, bd_id,
                 unkMacUcastAct=FLOOD_PROXY[
                     self.default_l2_unknown_unicast == 'flood' or
                     allow_broadcast],
-                unicastRoute=YES_NO[unicast_route],
+                unicastRoute=YES_NO[self.default_unicast_routing],
```

Comment:
This change breaks backward compatibility. Users of this API (ensure_bd_created_on_apic) expect to be able to control unicast routing through the unicast_route parameter which now is not used.
Also, we should not encourage more configuration options if not really necessary. Can the apicapi user decide whether or not unicast_route should be set to True or False based on its own internal configuration?